shuffle.js

```diff
@@ -0,0 +1,13 @@
+function shake(){
+  let abcs = 'abcdefghijklmnopqrstuvwxyz'
+  let letterChoice = '';
+  for(let i = 0; i < 16; i++){
+    let randomLetter = abcs.charAt(Math.floor(Math.random() * abcs.length));
```

Comment:
I'd write a function called, say, `randomElement(alphabet)` that encodes this logic. You can index into strings the same way you can arrays, so, e.g., `alphabet[2]` is `'c'`.